client/src/components/About.tsx

```diff
@@ -13,41 +21,95 @@ export const About: React.StatelessComponent = (): JSX.Element => (
             <h3 className="home-1">MyCourseIndex</h3>
             <h3 className="home-2">Search</h3>
         </Link>
-        <div className="center">
+        <div className="centerAbout">
             <h4>About</h4>
-            <h4><a href="https://github.com/oscarso2000/mycourseindex">Source code on Github</a></h4>
-            <p> The authors of the MyCourseIndex project are: </p>
 
-            <div className="item">
-                <img src={Magd} />
-                <span className="caption"><a href="https://github.com/bayoumi17m">Magd Bayoumi (mb2363)</a></span>
+            <p>MyCourseIndex was initially a project started for CS/INFO 4300: Language and Information that acts as an essential search engine for Cornell students and their courses. This search gathers all information from Piazza posts to Textbook and Notes Resources and returns valid results for the student to utilize. It is now being maintained and improved by members of <a href="https://cornelldata.science">Cornell Data Science Project Team</a>.</p>
```

Comment:
This line is much too long. It needs to be broken into broken down so that it's readable within an editor (without wrapping).

We also should update the description with more about the new product and less about the class etc. We want to focus on the product and team as much as possible

Reply:
Should i just hit enter at the end of every 130 characters or something since its wrapped in the p-text body already? 

Yea I just want the pictures up and running, we can change the description anytime.

Reply:
Ok, if that's the case, we can revisit this later